raw-svo.valid.main.lua

```diff
@@ -4544,12 +4544,23 @@ function valid.devastate_arms_cripple()
 end
 
 -- happens on wrist fracture levels 4,5
+-- edit: This ability can also mutilate a mangled limb.
 function valid.devastate_arms_mangle()
   checkaction(dict.wristfractures.gone, true)
   lifevision.add(actions.wristfractures_gone.p)
 
-  valid.simplemangledrightarm()
-  valid.simplemangledleftarm()
+  if affs.mangledrightarm then
+    removeaff("mangledrightarm")
```

Comment:
removeaff should be called once the line was validated, otherwise you'll fall for the illusion (and even the lifevision skill will be ignored) - so this should be done within svo.dict